### PR TITLE
In portal feedback UI

### PR DESCRIPTION
**Pull request by Bianca-Morris**
**Changelog:**
- Adds a button to the navbar that opens up a mailto link with some useful information pre-populated for submitting jira tickets.
- Deployed on cgap-devtest for testing

**Screenshots**
Button Placement:
[Image from Gyazo](https://gyazo.com/e9b1a4cf546cebf7c79a286aa8dbe7e7)
Options:
[Image from Gyazo](https://gyazo.com/1f0b05519321814a1fd380b281e17e44)
Example Email:
[Image from Gyazo](https://gyazo.com/2de456052770b4d4a1516a82c1154077)
Visible from Home Page:
[Image from Gyazo](https://gyazo.com/55bd1103dbbb25599a3d698bbae7d73b)

### CODE REVIEW

**Comment by Bianca-Morris** [comment]
Just a reminder to @alexkb0009 to review this when you get a chance.

**Comment by utku-ozturk** [comment]
I didn't test it locally but looks good to me! On the other hand, setting `cgap-support@hms-dbmi.atlassian.net` as an environment variable might be more flexible since multiple CGAP deployments may have their dedicated support in the future.

File: src/encoded/static/components/item-pages/components/FeedbackButton.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+Expected%20Behavior%3A%0D%0A%0D%0A%0D%0A\
+Actual%20Behavior%3A%0D%0A%0D%0A%0D%0A\
+Steps%20to%20Reproduce%20the%20Problem%3A%0D%0A%0D%0A%0D%0A`;
+
+const FEEDBACK_MAILTO = `mailto:cgap-support@hms-dbmi.atlassian.net?\
+subject=CGAP%20Feedback\
+&body=Feedback%20\
+Type%3A%20General%20Feedback%0D%0A%0D%0A\
+Comments%3A%0D%0A%0D%0A%0D%0A`;
+
```
Comment (dmichaels-harvard) [inline src/encoded/static/components/item-pages/components/FeedbackButton.js:27]: Very minor - might consider using encodeURIComponent on the above string so you don't have to have literal URL-encoded (e.g. %20) values.

**Review by shannonehmsen — COMMENTED** [review]
The functionality of the button is as expected.  I like the format of the email templates and information about the browser for context.

I am wondering if there is a way to have the buttons link to email in the browser or a page with submissions without opening the mail app on the computer. If this is a difficult add, then I think we can merge and come back to it later.

Feedback Button

For styling, it would be great to give some breathing room between the question mark icon, title and arrow. Other than that, looks good to me.

**Comment by Bianca-Morris** [comment]
> The functionality of the button is as expected. I like the format of the email templates and information about the browser for context.
> 
> I am wondering if there is a way to have the buttons link to email in the browser or a page with submissions without opening the mail app on the computer. If this is a difficult add, then I think we can merge and come back to it later.

There is not a great way to do that. We could link directly to the help desk's web form, but we wouldn't be able to pre-populate the fields.

> Feedback Button
> 
> For styling, it would be great to give some breathing room between the question mark icon, title and arrow. Other than that, looks good to me.

Sure, I can see about adding some padding.

**Pull request merged**